jaqpotpy/descriptors/molecular/pub_chem_fingerprint.py

```diff
@@ -72,3 +72,13 @@ def _featurize(self, datapoint: RDKitMol, **kwargs) -> np.ndarray:
         pubchem_compound = pcp.get_compounds(smiles, 'smiles')[0]
         feature = [int(bit) for bit in pubchem_compound.cactvs_fingerprint]
         return np.asarray(feature)
+    
+    def _get_column_names(self, **kwargs) -> list:
+        descriptors = []
+        for i in range(881):
```

Comment:
Why is the range here hardcoded to 881?

Reply:
It is the length of the vector that this featurizer generates for each SMILES, so it is constant.

Reply:
Then I suggest adding a constant variable at the top of the file just after the imports e.g. 
```python
PUBCHEM_VECTOR_LENGTH = 881
``` 

and use it like this
```suggestion
        for i in range(PUBCHEM_VECTOR_LENGTH):
```